crates/polars-core/src/frame/row/dataframe.rs

```diff
@@ -40,65 +40,36 @@ impl DataFrame {
     /// This should only be used when you have row wise data, as this is a lot slower
     /// than creating the [`Series`] in a columnar fashion
     pub fn from_rows_and_schema(rows: &[Row], schema: &Schema) -> PolarsResult<Self> {
-        Self::from_rows_iter_and_schema(rows.iter(), schema)
+        Self::from_rows_iter_and_schema(rows.iter().cloned(), schema)
```

Comment:
This now clones every row we add? I am for making this more generic, but we should not clone vec buffers for every row.

Reply:
Apologies for the late reply.

Thank you for your suggestion. I've fixed it. Could you please take another look and let me know if it's acceptable now?